### PR TITLE
Change get started links

### PR DESCRIPTION
**Pull request by bmckercher123**
Add code to make external links to open in a new window for Get started.

- Merge to nightly

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit f57e6718466ba5536c6c0f0bcfec48283fd09cb2

https://deploy-preview-3278--calico.netlify.com

File: getting-started/kubernetes/self-managed-public-cloud/gce.md [hunk 1]
```diff
@@ -179,14 +179,14 @@ The geeky details of what you get:
...
+{% include open-new-window.html text='Kubespray' url='https://kubespray.io/' %} is a tool for provisioning and managing Kubernetes clusters with support for multiple clouds including Google Compute Engine.  Calico is the default networking provider, or you can set the `kube_network_plugin` variable to `calico`. See the Kubespray docs for more details. See the {% include open-new-window.html text='Kubespray docs' url='https://kubespray.io/#/?id=network-plugins' %} for more details.
 
 ### Above and beyond
 
-- [Video: Everything you need to know about Kubernetes networking on Google cloud](https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-networking-on-google-cloud/)
+- {% include open-new-window.html text='[Video: Everything you need to know about Kubernetes networking on Google cloud' url='https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-networking-on-google-cloud/' %}
```
Comment (lxpollitt) [inline getting-started/kubernetes/self-managed-public-cloud/gce.md]: Typo: `[Video` -> `Video`

File: getting-started/openshift/installation.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Install an OpenShift v4 cluster with {{site.prodname}}.
 
 ### Value
 
-Augments the applicable steps in the [OpenShift documentation](https://cloud.redhat.com/openshift/install)
+Augments the applicable steps in the {% include open-new-window.html text='OpenShift documentation' url='https://cloud.redhat.com/openshift/install' %}
```
Comment (lxpollitt) [inline getting-started/openshift/installation.md]: I wonder if this should link to: https://docs.openshift.com/. (The existing link asks me to sign into redhat, so I can't tell what content it actually ends up with.)

Comment (lxpollitt) [reply getting-started/openshift/installation.md]: One of our regular openshift users in eng might know.

File: getting-started/openshift/installation.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Now run OpenShift installer to create a default configuration file:
 openshift-install create install-config
 ```
 
-> **Note**: Refer to the OpenShift installer documentation found on [https://cloud.redhat.com/openshift/install](https://cloud.redhat.com/openshift/install) for more information
+> **Note**: Refer to the {% include open-new-window.html text='OpenShift installer documentation' url='https://cloud.redhat.com/openshift/install](https://cloud.redhat.com/openshift/install' %} for more information
```
Comment (lxpollitt) [inline getting-started/openshift/installation.md]: Malformed url parameter.

File: getting-started/kubernetes/quickstart.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ To deploy a cluster suitable for production, refer to [Installation](installatio
   If NetworkManager is present on the host, refer to
   [Configure NetworkManager](../../maintenance/troubleshooting#configure-networkmanager).
 
-- [Follow the Kubernetes instructions to install kubeadm](https://kubernetes.io/docs/setup/independent/install-kubeadm/){:target="_blank"}.
+- {% include open-new-window.html text='Follow the Kubernetes instructions to install kubeadm' url='https://kubernetes.io/docs/setup/independent/install-kubeadm/' %}
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply getting-started/kubernetes/quickstart.md]: Fixed

File: getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ This how-to guide uses the following {{site.prodname}} features:
 ### Before you begin...
 
 - Ensure that your Kubernetes cluster meets [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements).
-  If you do not have a cluster, see [Using kubeadmin to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+  If you do not have a cluster, see {% include open-new-window.html text='Using kubeadmin to create a cluster' url='http://kubernetes.io/docs/getting-started-guides/kubeadm/' %}.
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-onprem/onpremises.md]: Fixed.

File: getting-started/kubernetes/self-managed-public-cloud/gce.md (hunk 1 above)
Comment (bmckercher123) [reply getting-started/kubernetes/self-managed-public-cloud/gce.md]: Fix

File: getting-started/openshift/installation.md (hunk 3 above)
Comment (bmckercher123) [reply getting-started/openshift/installation.md]: fix

**Pull request merged**